### ISSUE TITLE
Cheerio dependancy error

### ISSUE THREAD

**Issue by sthompson232** [evecalm/truncate-html#39]
I have suddenly started receiving the following error when installing `truncate-html` latest version. This looks like a dependancy update error from cheerio. Perhaps cheerio needs to be pinned at a specific version.

```
[ERROR] No matching export in "node_modules/cheerio/dist/browser/index.js" for import "default"

    node_modules/truncate-html/dist/truncate.es.js:1:7:
      1 │ import $e9N8g$cheerio from "cheerio";
        ╵        ~~~~~~~~~~~~~~


 ERROR  error while updating dependencies:
Error: Build failed with 1 error:
node_modules/truncate-html/dist/truncate.es.js:1:7: ERROR: No matching export in "node_modules/cheerio/dist/browser/index.js" for import "default"
```

**Comment by oe** [comment]
Cheerio has released stable v1.0 a few days ago, but its APIs are slightly different from the latest beta version `1.0.0-rc.12`.

I'm working on it now, but before that you can override the cheerio version to fix it.

Override version by adding fields to package.json.

If you are using npm
```json
{
  ...
  "overrides": {
    "cheerio": "1.0.0-rc.12"
  },
  ...
}
```

If you are using pnpm
```json
{
  ...
  "pnpm": {
    "overrides": {
      "cheerio": "1.0.0-rc.12"
    }
  },
  ...
}
```

if you are using yarn
```json
{
  ...
  "resolutions": {
    "cheerio": "1.0.0-rc.12"
  },
  ...
}
```

**Comment by oe** [comment]
`truncate-html@1.1.2` has just been released, cheerio has been pinned to `1.0.0-rc.12`.
